dist/api_data.py

```diff
@@ -1739,6 +1739,11 @@ def __ge__(self, other):
                 Config('visible_only', 'false', r'''
                     only dump updates that are visible to the session''',
                     type='boolean', undoc=True),
+                Config('end_timestamp', '', r'''
```

Comment:
Wouldn't it be more appropriate for this to be named `start_timestamp`? Since it's only returning updates *newer* than this, at least from my reading of the description.

Reply:
yes, that's better